server.js

```diff
@@ -338,8 +338,13 @@ module.exports.initExpress = function () {
   app.use('/pl/workspace/:workspace_id/container', [
     cookieParser(),
     (req, res, next) => {
-      // Needed for workspaceAuthRouter and `selectAndValidateWorkspace` middleware.
+      // Needed for workspaceAuthRouter.
       res.locals.workspace_id = req.params.workspace_id;
+
+      // Needed for logs; this will retain the original URL if we end up
+      // rewriting the URL in `pathRewrite` above.
+      req.originalUrl = req.url;
```

Comment:
I thought express automatically set `req.originalUrl` for us? (see [the docs](https://expressjs.com/en/api.html#req.originalUrl))

Reply:
Ah, true! Good catch.

Reply:
Resolved in 2fe82a9.